proto/process/agent.proto

```diff
@@ -527,7 +535,8 @@ message Connection {
 	ConnectionFamily family = 10;
 	ConnectionType type = 11;
 	int64 pidCreateTime = 12;
-
+	EphemeralPortState isEphemeralPort = 41;
```

Comment:
Do you think it makes more sense to have this flag be part of `Addr`, named something like `ephemeralPort`? I realize that we won't ever know the value for `raddr`.

Otherwise, I would rename this to something like `ephemeralLAddrPort` to make it clear which port/addr the flag is for.